tests/console/verify_efi_mok.pm

```diff
@@ -140,24 +141,23 @@ sub get_esp_info {
         svirt_hyperv => 'scsi'
     };
     # return a the first element (drive or partition number) from parted's output
```

Comment:
Comment doesn't match the code

Reply:
thanks, that was a left over including the above hash